README.md

```diff
@@ -1 +1,150 @@
+      <br>Node.js
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/mongodb/mongodb-original.svg" alt="MongoDB" />
+      <br>MongoDB
+    </td>
+    <td align="center">
+      <img width="80px" src="https://upload.wikimedia.org/wikipedia/commons/a/ab/Swagger-logo.png?20170812110931" alt="Swagger" />
+      <br>Swagger
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/express/express-original-wordmark.svg" alt="Express.js" />
+      <br>Express.js
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/jest/jest-plain.svg" alt="Jest" />
+      <br>Jest
+    </td>
+    <td align="center">
+      <img width="80px" src="https://repository-images.githubusercontent.com/1272424/d1995000-0ab7-11ea-8ed3-04a082c36b0d" alt="nodemailer" />
+      <br>Nodemailer
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/typescript/typescript-original.svg" alt="TypeScript" />
+      <br>TypeScript
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/tailwindcss/tailwindcss-plain.svg" alt="Tailwind CSS" />
+      <br>Tailwind CSS
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/vscode/vscode-original.svg" alt="vscode" />
+      <br>Vscode
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original.svg" alt="git" />
+      <br>Git
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.worldvectorlogo.com/logos/jwt-3.svg" alt="jwt" />
+      <br>JWT
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg" alt="github" />
+      <br>Github
+    </td>
+  </tr>
+</table>
+
+
+## Project Features
+
+- Home Page: Displaying information about the project including project overview, team members, and project design files.
+- Contact-us route: A way for users to get in touch with the platform administrators by sending a message to the admin email.
+- User Authentication: Secure signup and login process with options for Google or sign-up/sign-in form.
+- User Profile: Users can manage their personal information, profile pictures, and provide a valid location in Turkey.
+- Dish Listings: Cooks can post dishes along with images, descriptions, categories, and allergy information.
+- Order Management: Users can browse dishes, place orders, and view their order history.
+- Cart: Users can add and remove items from their cart before finalizing orders.
+- Admin Dashboard: Cooks and Admin have extra access for managing dishes and orders.
```

Comment:
In here maybe we can write in this way:
 Admin Dashboard: Admin has extra access for users, dishes and orders.

Reply:
✅